verilog/formatting/tree_unwrapper.cc

```diff
@@ -1321,6 +1376,49 @@ void TreeUnwrapper::ReshapeTokenPartitions(
       break;
     }
 
+    case NodeEnum::kRandomizeMethodCallExtension:
+    case NodeEnum::kRandomizeFunctionCall: {
+      auto& last = *ABSL_DIE_IF_NULL(partition.RightmostDescendant());
+      if (PartitionIsCloseParen(last) || PartitionStartsWithSemicolon(last)) {
+        verible::MergeLeafIntoPreviousLeaf(&last);
+      }
+
+      auto& uwline = partition.Value();
+      const auto partition_policy = uwline.PartitionPolicy();
+      if (partition_policy == PartitionPolicyEnum::kAppendFittingSubPartitions) {
+        if (partition.Children().size() > 2) {
+          uwline.SetPartitionPolicy(PartitionPolicyEnum::kAlwaysExpand);
+        }
+      }
+      break;
+    }
+
+    case NodeEnum::kMethodCallExtension:
+    case NodeEnum::kSystemTFCall:
+    case NodeEnum::kFunctionCall: {
+      auto& last = *ABSL_DIE_IF_NULL(partition.RightmostDescendant());
+      if (PartitionIsCloseParen(last) || PartitionStartsWithSemicolon(last)) {
+        verible::MergeLeafIntoPreviousLeaf(&last);
+      }
+      break;
+    }
+
+    case NodeEnum::kIfHeader: {
+      if (partition.Children().size() > 1) {
+        const auto& uwline = partition.Children()[1].Value();
```

Comment:
Use one of the functions in 
https://github.com/google/verible/blob/master/verilog/CST/statement.h
for accessing substructure of `kIfHeader` nodes (instead of `[1]`).

Reply:
I access here UnwrapperLine child, not a CST node. Does CST accessors still apply here?

Reply:
Sorry, these are token partitions already, and there are no equivalent functions because the structure is fluid.
Worth adding a comment about what `[1]` is intended to reference.